.github/workflows/ccip-live-network-tests.yml

```diff
@@ -0,0 +1,306 @@
+name: CCIP On-Demand Live Network Tests
```

Comment:
This workflow had recent key changes in latest ccip-develop branch. Need to port that changes here.

Reply:
Done